src/editor/modules/image.vue

```diff
@@ -34,7 +35,7 @@ export default {
         dropzoneOptions () {
           return {
             // custom dropzone options
-            ...this.options.image.dropzoneOptions,
+            options: this.options.image.dropzoneOptions,
```

Comment:
I believe this conflicts with https://github.com/chmln/vue-wysiwyg/pull/58. Could you check that this doesn't break dropzone options?